ui/src/variables/selectors/index.tsx

```diff
@@ -15,29 +15,28 @@ import {
 import {VariableAssignment} from 'src/types/ast'
 import {
   AppState,
+  ResourceState,
   VariableArguments,
   VariableArgumentType,
   Variable,
-} from 'src/types'
-import {
   VariableValues,
   VariableValuesByID,
   ValueSelections,
-} from 'src/variables/types'
+} from 'src/types'
 
-type VariablesState = AppState['variables']['variables']
-type ValuesState = AppState['variables']['values']['contextID']
+type VariablesState = ResourceState['variables']['byID']
+type ValuesState = ResourceState['variables']['values']['contextID']
 
 const extractVariablesListMemoized = memoizeOne(
   (variablesState: VariablesState): Variable[] => {
-    return Object.values(variablesState)
-      .filter(d => d.status === RemoteDataState.Done)
-      .map(d => d.variable)
+    return Object.values(variablesState).filter(
+      d => d.status === RemoteDataState.Done
```

Comment:
what is `d`?

Reply:
idk.  it was there  when i got there.

Reply:
idk `d` was there when i got there.  😄

Reply:
this is how i feel about little things like this:

[>don't ya see it? well pick it up!!](https://www.youtube.com/watch?v=wwQ5SmhwGgk&feature=youtu.be&t=156)

basically it's all our job to sweep the floor and we should keep the place tidy for the next person. anything we can do to reduce our collective mental load. to that point, maybe `data` is slightly better than `d`?